packages/service-clients/odsp-client/src/test/odspTestTokenProvider.ts

```diff
@@ -36,12 +37,18 @@ export class OdspTestTokenProvider implements IOdspTokenProvider {
 		const sharePointScopes = `${siteUrl}/Container.Selected`;
 		const tokens = await this.fetchTokens(siteUrl, sharePointScopes);
 		return {
-			fromCache: true,
+			fromCache: false,
 			token: tokens.accessToken,
 		};
 	}
 
-	private async fetchTokens(siteUrl: string, scope: string) {
+	private async fetchTokens(
+		siteUrl: string,
+		scope: string,
+	): Promise<{
+		accessToken: string;
+		refreshToken?: string;
```

Comment:
@sonalideshpandemsft we weren't using this result, so I typed it as optional, and only validate the contents if they're present in the response. Let me know if that doesn't make sense.